compute_tools/src/bin/fast_import.rs

```diff
@@ -83,6 +87,189 @@ const DEFAULT_LOCALE: &str = if cfg!(target_os = "macos") {
     "C.UTF-8"
 };
 
+async fn decode_connstring(
+    kms_client: &aws_sdk_kms::Client,
+    key_id: &String,
+    connstring_ciphertext_base64: Vec<u8>,
+) -> Result<String, anyhow::Error> {
+    let mut output = kms_client
+        .decrypt()
+        .key_id(key_id)
+        .ciphertext_blob(aws_sdk_s3::primitives::Blob::new(
+            connstring_ciphertext_base64,
+        ))
+        .send()
+        .await
+        .context("decrypt connection string")?;
+
+    let plaintext = output
+        .plaintext
+        .take()
+        .context("get plaintext connection string")?;
+
+    String::from_utf8(plaintext.into_inner()).context("parse connection string as utf8")
+}
+
+struct PostgresProcess {
```

Comment:
nit: This can be in a separate module(file).

Reply:
idk, I would keep it in one file as long as it is readable, will keep it this way for now